.wp-env.json

```diff
@@ -1,6 +1,8 @@
 {
 	"core": "WordPress/WordPress",
-	"plugins": [ "." ],
+	"plugins": [
+		"."
+	],
```

Comment:
Seems unrelated?

Reply:
Yes, was testing it with `wp-env` and in retrospect, I should use `.wp-env.override.json` 😅

Reply:
It's VSC on save refactor, I had to fix it with the terminal :)